jenkins-jobs/docker/debezium-testing-opneshift/testsuite-deployment.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+. /testsuite/library.sh
```

Comment:
prefer  ``source`` over execution  for example new environment variables would not to be propagated back to the original shell)